__tests__/plugins/formatters.commerce.test.ts

```diff
@@ -71,12 +71,6 @@ loader.paths('f-product-quick-view-%N.html').forEach((path) => {
   test(`product quick view - ${path}`, () => loader.execute(path));
```

Comment:
All of the new tests in this file and all of the related test files were ported over from template-compiler ([tests](https://github.com/Squarespace/template-compiler/blob/main/core/src/test/java/com/squarespace/template/plugins/platform/CommerceFormattersTest.java/#L105-L159), [test templates](https://github.com/Squarespace/template-compiler/tree/main/core/src/test/resources/com/squarespace/template/plugins/platform))